README.md

```diff
@@ -65,6 +65,32 @@
 
 ______________________________________________________________________
 
+## Highlights
+
+- Works out-of-the-box
+  - all that's needed is to [install `scipy-stubs`](#installation)
+  - does not require a `mypy` plugin or other configuration
+  - available on [PyPI](https://pypi.org/project/scipy-stubs/) and [Anaconda](https://anaconda.org/conda-forge/scipy-stubs)
```

Comment:
It is more accurate to say conda-forge here - you can access conda-forge packages independently from anaconda.org, for example: https://prefix.dev/channels/conda-forge/packages/scipy-stubs. Probably fine to still link to the anaconda.org page but either would do.

```suggestion
  - available on [PyPI](https://pypi.org/project/scipy-stubs/) and [conda-forge](https://anaconda.org/conda-forge/scipy-stubs)
```